src/features/formData/useFormDataQuery.tsx

```diff
@@ -15,6 +15,17 @@ export function useFormDataQueryDef(url: string | undefined): QueryDefinition<un
   const { fetchFormData } = useAppQueries();
   const queryKey = useFormDataQueryKey(url);
   const options = useFormDataQueryOptions();
+  const isStateless = useApplicationMetadata().isStatelessApp;
+
+  if (isStateless) {
+    return {
+      queryKey,
+      queryFn: url ? () => fetchFormData(url, options) : skipToken,
+      enabled: true,
+      gcTime: 0,
```

Comment:
Skal ikke hindre merge, men bare noen tanker 😄 

Kunne det vært nyttig å sette `staleTime` i stedet for `gcTime` for at senere fetches skal hente med stale-while-revalidate? 

En annen tanke er også at `isStateless`-variabelen har noe å si hvor hva som returneres, og burde da kanskje være i `queryKey`-en? Typ `[...queryKey, { isStateless }]` eller noe sånn at man har forskjellig queryKey for stateful og stateless.

Reply:
Hvis vi har har stale-while-revalidate, vil vi ikke da vise gammel data til vi har fått ny data? Det vil vi ikke i det tilfellet appen kommer fra ihvertfall siden de bruker denne data til å vise om man har startet på skjemaet eller ikke. Vi ville da vist siden med "start nytt skjema" først, og så "Fortsett på skjema" når dataen er lastet.

Litt usikker på hva vi vinner på å ha isStateless i queryKey? At vi trigger query når isStateless forandrer seg?

Reply:
Query keyen bør allerede være forskjellig når man er i stateless siden query keyen er basert på url-en den henter ifra. For stateful er det url til data-elementet, men for stateless er det en annen url som refererer til data-typen som brukes i stateless (siden det ikke finnes noe data element da)